functional/util.py

```diff
@@ -91,6 +103,77 @@ def is_iterable(val):
     return isinstance(val, collections.Iterable)
 
 
+def split_every(parts, iterable):
+    """
+    Split an iterable in n parts
+
+    >>> l = [1, 2, 3, 4]
+    >>> split_every(l, 2)
+    [[1, 2], [3, 4]]
```

Comment:
This can also very easily be made into a unit test on its own (keep the doctest/example though)